### DISCUSSION TITLE
LocalizationKey in SpeechCommand

### DISCUSSION THREAD

**Discussion by ag-kfe** [microsoft/MixedRealityToolkit-Unity#9625] [Q&A]
Does anyone know what the Localization key does and how to use it?

when i try to enter a text i get the following exception. 

_Exception thrown at 0x00007FF80124A71C (KernelBase.dll) in SpeechTest.exe: WinRT originate error - 0x80073B1F : 'ResourceMap Not Found.'.
onecoreuap\base\mrt\runtime\com\winrt\core\winrtresourceloader.cpp(114)\MrmCoreR.dll!00007FFFFAFF7ADC: (caller: 00007FFFFAFCDFB0) ReturnHr(6) tid(a1c) 80073B1F ResourceMap Not Found._

And the description in the docu doesent help. 
_"Add a localization key that will be used in UWP apps to obtain the localized keyword from the app resources."_

I use 
Unity 2019 
MRTK 2.5 
IL2CPP

![Localizationkey](https://user-images.githubusercontent.com/66006099/113729585-704b3e80-96f7-11eb-8228-49c2377f5882.PNG)

**Comment by polar-kev** [comment]
Hi @agkfe, thanks for flagging this. It looks like we have a documentation gap. I will convert this to an issue so we can track and resolve it.
